Source/WPEFramework/SystemInfo.h

```diff
@@ -27,11 +27,124 @@ namespace WPEFramework {
 namespace PluginHost {
 
     class SystemInfo : public PluginHost::ISubSystem {
-    private:
+    public:
         SystemInfo() = delete;
         SystemInfo(const SystemInfo&) = delete;
         SystemInfo& operator=(const SystemInfo&) = delete;
 
+    private:
```

Comment:
Looks like this is a copy of the core, why not just use the core functionality, why copy it here ?

Reply:
Moved this to DeviceIdentification plugin